src/content/en/2021/performance.md

```diff
@@ -21,11 +21,11 @@ unedited: true
 
 ## Introduction
 
-Performance is important for user experience. Slow-loading and slow-to-respond websites frustrate users and cause lost conversions. This is the first year that the <a hreflang="en" href="https://web.dev/vitals/">Core Web Vitals</a> have contributed to Google search rankings. As such, we've seen greater interest in  improving website performance which is great news for users.
+Performance is important for the user experience. Slow-to-load and slow-to-respond websites frustrate users and cause lost conversions. This is the first year that <a hreflang="en" href="https://web.dev/vitals/">Core Web Vitals</a> have contributed to Google search rankings. As such, we've seen greater interest in improving website performance which is great news for users.
```

Comment:
I think "the user experience" sounds awkward. I don't think it's used that way in web dev conversations usually.

Reply:
I put "the user experience" because that's how it's talked about in the UI/UX circles.

Reply:
Let me clarify that most of it is not "talked about" but written.

Reply:
Ex: mostly in the titles though: https://www.usability.gov/what-and-why/user-experience.html

Reply:
https://twitter.com/TheGreenGreek/status/1459882273415835648